blockly/apps/blocklyduino/index.html

```diff
@@ -150,7 +150,12 @@
 
   var toolbox = document.getElementById('toolbox');
   Blockly.inject(document.getElementById('content_blocks'),
-      {media: '../../media/',
+      {grid:
```

Comment:
I've added a grid to the workspace (now Blockly feature).  Feel free to remove if you don't like it.